### PR TITLE
feat(expressive modal): v1 & v2 guidance updates

### PR DESCRIPTION
**Pull request by sangeethababu9223**
### Related Ticket(s)

https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/issues/1616

### Description

v1 & v2 guidance updates

### Changelog

**New**

- Added behaviour section with details on Model position, Overflow and closing the modal 

**Changed**

- Updated the default use case image with one from lightbox media viewer

**Removed**

- Removed use case image for large variant

### CODE REVIEW

File: src/pages/components/expressive-modal.mdx [hunk 1]
```diff
@@ -34,7 +33,13 @@ The default variation reflects the <a href="https://www.carbondesignsystem.com/c
 
 <Title>Use case</Title>
 
-![Image of Expressive modal used in a login context](../../images/component/expressive-modal/expressive-modal-default--example.jpg)
+<Row>
+<Column colMd={8} colLg={8}>
+
+![Image of Expressive modal used in a login context](../../images/component/expressive-modal/expressive-modal-default--usecase.jpg)
```
Comment (ariellalgilmore) [inline src/pages/components/expressive-modal.mdx:39]:
```suggestion
![Image of Expressive modal with heading and copy descriping the media](../../images/component/expressive-modal/expressive-modal-default--usecase.jpg)
```

File: src/pages/components/expressive-modal.mdx [hunk 2]
```diff
@@ -50,8 +55,21 @@ The expanded variation fills the screen and is typically used to display large a
...
-![Image of Expressive modal expanded used to illustrate the IBM design thinking process](../../images/component/expressive-modal/expressive-modal-expanded--example.jpg)
+### Overflow
 
+When the modal content is longer than the modal height then the body section should scroll vertically.
+The content should visibly fade at the end of the modal body area to indicate there is additional content out of view.
+Modal content should never scroll horizontally; instead, use a larger size modal.
+
+### Closing the modal
+
+Clicking on the close icon x or on the overlay will dismiss the modal.
```
Comment (ariellalgilmore) [inline src/pages/components/expressive-modal.mdx:72]:
```suggestion
Clicking on the close button icon [ x ] or on the dark overlay will dismiss the modal.
```

**Review by ariellalgilmore — COMMENTED** [review]
Looking good!!! just a few small comments. I also wonder if we can add any additional images/gifs in the behavior section showing the scrolling or closing of the modal

File: src/pages/components/expressive-modal.mdx (hunk 2 above)
Comment (sangeethababu9223) [reply src/pages/components/expressive-modal.mdx]: Done

File: src/pages/components/expressive-modal.mdx (hunk 1 above)
Comment (sangeethababu9223) [reply src/pages/components/expressive-modal.mdx]: Done

**Comment by sangeethababu9223** [comment]
@ariellalgilmore ,
Added images to the behaviour section.

**Review by oliviaflory — APPROVED** [review]
This looks great, thank you @sangeethababu9223!

**Comment by oliviaflory** [comment]
@sangeethababu9223 could you please look into adding the GIF player to your two GIFs? I think the `light` prop might be best but if you could try light for the first and `dark` for the second so we can see both that would be helpful. 

We could take a static pull from your video files for the static image?

**Comment by sangeethababu9223** [comment]
@oliviaflory,
Added Gif player as per your comment. Please check.

**Comment by oliviaflory** [comment]
It's perfect, thanks you @sangeethababu9223 !

**Review by oliviaflory — APPROVED** [review]

**Pull request merged**